src/pages/public_registration/ActionPanel.tsx

```diff
@@ -19,23 +20,61 @@ export const ActionPanel = ({
   isLoadingData,
 }: ActionPanelProps) => {
   const { t } = useTranslation();
+  const [tabValue, setTabValue] = useState(0);
 
   return (
-    <Paper
-      elevation={0}
-      data-testid={dataTestId.registrationLandingPage.tasksPanel.panelRoot}
-      sx={{ gridArea: 'tasks' }}>
-      <StyledPaperHeader>
-        <Typography color="inherit" variant="h1">
-          {t('common.tasks')}
-        </Typography>
-      </StyledPaperHeader>
-      <ActionPanelContent
-        tickets={tickets}
-        refetchData={refetchRegistrationAndTickets}
-        isLoadingData={isLoadingData}
-        registration={registration}
-      />
+    <Paper elevation={0} data-testid={dataTestId.registrationLandingPage.tasksPanel.panelRoot}>
+      <Box sx={{ color: 'primary.contrastText' }}>
```

Comment:
Måtte legge til `color` på parent fordi `textColor` på `Tabs` aksepterer kun `'primary',` `'secondary'` og `'inherit'`.

Reply:
Om Box her brukes bare for å sette color, kan sikkert boxen fjernes, og `sx` flyttes til `Paper` over?